src/game/camera.c

```diff
@@ -4741,6 +4881,7 @@ void handle_c_button_movement(struct Camera *c) {
     }
 }
 
+
```

Comment:
get this formatting change outta here!!!!!!

Reply:
I think arceveti took over my pc